src/ssl.c

```diff
@@ -40427,10 +40565,66 @@ void wolfSSL_sk_X509_NAME_free(WOLF_STACK_OF(WOLFSSL_X509_NAME)* sk)
     XFREE(sk, sk->heap, DYNAMIC_TYPE_OPENSSL);
 }
 
+#if defined(WOLFSSL_APACHE_HTTPD) || defined(OPENSSL_ALL)
+/* Helper function for X509_NAME_print_ex. Sets *buf to string for domain
+   name attribute based on NID. Returns size of buf */
+static int get_dn_attr_by_nid(int n, char** buf)
```

Comment:
Please eliminate cast and use `const char** buf` to avoid any possible future issues or confusion.